addons/paint-snap/userstyle.css

```diff
@@ -2,13 +2,13 @@
   position: relative;
 }
 .sa-paint-snap-button:focus-within {
-  background-color: hsla(215, 100%, 65%, 0.2);
+  background-color: hsla(260, 60%, 60%, 0.2);
```

Comment:
Editor dark mode and customizable colors doesn't affect this color, but it probably should.
![Input with purple background next to red buttons](https://github.com/ScratchAddons/ScratchAddons/assets/106490990/7e72de3c-98a1-4d81-b262-aed7f0754280)